x-pack/plugins/actions/server/saved_objects/dashboard/dashboard.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { DashboardAttributes } from '@kbn/dashboard-plugin/common';
+import { SavedObject } from '@kbn/core-saved-objects-common/src/server_types';
+
+export const genAiSavedObjectId = 'f23c4320-00aa-11ee-a91d-genaisoid088';
```

Comment:
It has conflict when installing to multiple spaces, might be better not to use a hard coded id:
<img width="1288" alt="conflict" src="https://github.com/elastic/kibana/assets/6295984/03423d55-17ce-4136-8104-837be2d1f75b">

Reply:
Hm. How should I check if it exists or not if we do not know the id? I can name something like `cool-id-${space}`. I'm also wondering if I should store this as a reference instead of on the config since it is a saved object id